docs/web-wallet.md

```diff
@@ -150,7 +150,7 @@ const verificationRequest = {
       circuitId: "credentialAtomicQuerySigV2",
       id: 1711399135,
       query: {
-        allowedIssuers: ["*"],
+        allowedIssuers: ["did:polygonid:polygon:amoy:2qV9QXdhXXmN5sKjN1YueMjxgRbnJcEGK2kGpvk3cq"],
```

Comment:
It should be did:iden3:privado:main:.... Ask guys for proper one.

Reply:
You can use this for example @0xpulkit:

```
did:iden3:privado:main:2ScrbEuw9jLXMapW3DELXBbDco5EURzJZRN1tYj7L7
```

Reply:
Sure, Let me ask

Reply:
have updated the verifier and the issuer did